src/test/resources/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs/expected/includeLowQualSites.vcf

```diff
@@ -0,0 +1,51 @@
+##INFO=<ID=AN,Number=1,Type=Integer,Description="Total number of alleles in called genotypes">
+##INFO=<ID=BaseQRankSum,Number=1,Type=Float,Description="Z-score from Wilcoxon rank sum test of Alt Vs. Ref base qualities">
+##INFO=<ID=ClippingRankSum,Number=1,Type=Float,Description="Z-score From Wilcoxon rank sum test of Alt vs. Ref number of hard clipped bases">
+##INFO=<ID=DP,Number=1,Type=Integer,Description="Approximate read depth; some reads may have been filtered">
+##INFO=<ID=DS,Number=0,Type=Flag,Description="Were any of the samples downsampled?">
+##INFO=<ID=END,Number=1,Type=Integer,Description="Stop position of the interval">
+##INFO=<ID=ExcessHet,Number=1,Type=Float,Description="Phred-scaled p-value for exact test of excess heterozygosity">
+##INFO=<ID=FS,Number=1,Type=Float,Description="Phred-scaled p-value using Fisher's exact test to detect strand bias">
+##INFO=<ID=HaplotypeScore,Number=1,Type=Float,Description="Consistency of the site with at most two segregating haplotypes">
+##INFO=<ID=InbreedingCoeff,Number=1,Type=Float,Description="Inbreeding coefficient as estimated from the genotype likelihoods per-sample when compared against the Hardy-Weinberg expectation">
+##INFO=<ID=MLEAC,Number=A,Type=Integer,Description="Maximum likelihood expectation (MLE) for the allele counts (not necessarily the same as the AC), for each ALT allele, in the same order as listed">
+##INFO=<ID=MLEAF,Number=A,Type=Float,Description="Maximum likelihood expectation (MLE) for the allele frequency (not necessarily the same as the AF), for each ALT allele, in the same order as listed">
+##INFO=<ID=MQ,Number=1,Type=Float,Description="RMS Mapping Quality">
+##INFO=<ID=MQRankSum,Number=1,Type=Float,Description="Z-score From Wilcoxon rank sum test of Alt vs. Ref read mapping qualities">
+##INFO=<ID=QD,Number=1,Type=Float,Description="Variant Confidence/Quality by Depth">
+##INFO=<ID=RAW_MQ,Number=1,Type=Float,Description="Raw data for RMS Mapping Quality">
+##INFO=<ID=ReadPosRankSum,Number=1,Type=Float,Description="Z-score from Wilcoxon rank sum test of Alt vs. Ref read position bias">
+##INFO=<ID=SOR,Number=1,Type=Float,Description="Symmetric Odds Ratio of 2x2 contingency table to detect strand bias">
+##contig=<ID=20,length=63025520,assembly=b37>
+##contig=<ID=21,length=48129895,assembly=b37>
+##reference=file:///Users/cnorman/projects/gatk/src/test/resources/large/human_g1k_v37.20.21.fasta
+##source=GenotypeGVCFs
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	GTEX-OXRP-0003	GTEX-QXCU-0004	GTEX-RVPV-0003
+20	10012730	.	G	.	27.02	.	DP=4	GT:AD:DP:RGQ	0/0:2,0:2:6	./.:0,0:0	0/0:2,0:2:6
+20	10012731	.	C	.	27.02	.	DP=4	GT:AD:DP:RGQ	0/0:2,0:2:6	./.:0,0:0	0/0:2,0:2:6
+20	10012732	.	A	.	27.02	.	DP=4	GT:AD:DP:RGQ	0/0:2,0:2:6	./.:0,0:0	0/0:2,0:2:6
+20	10012733	.	G	.	27.02	.	DP=4	GT:AD:DP:RGQ	0/0:2,0:2:6	./.:0,0:0	0/0:2,0:2:6
+20	10012734	.	G	.	0.63	LowQual	DP=4;ExcessHet=3.01;MQ=NaN	GT:DP:RGQ	0/0:2:6	0/0:0:0	0/0:2:6
```

Comment:
I looked at the input for this and there were PLs for all the samples (and a bunch more FORMAT annotations):
`20	10012751	.	T	C,<NON_REF>	.	.	DP=4;ExcessHet=3.01;RAW_MQ=14400.00	GT:AD:DP:GQ:MIN_DP:PL:SB	./.:0,2,0:2:6:.:49,6,0,49,6,49:0,0,1,1	./.:.:0:0:0:0,0,0,0,0,0	./.:0,2,0:2:6:.:49,6,0,49,6,49:0,0,1,1`
Specifically that first sample should be homVar (1/1) even though it's GQ is pretty crappy.  If they're all actually homRef then it would be very strange to call a variant there, even if it's very low quality.  Am I looking at the wrong input?

Reply:
Missing ADs are also why the MQ=NaN.

Reply:
Totally looking at the wrong input.  Sorry.  I do wonder about the ADs, though it looks like that was also an issue in the GATK3 output.